src/PairedRouting.php

```diff
@@ -772,6 +772,45 @@ public function add_permalink_settings_notice() {
 		<?php
 	}
 
+	/**
+	 * Update canonical URL for legacy reader paired urls.
+	 *
+	 * @param string $redirect_url Canonical redirect URL.
+	 *
+	 * @return string Updated canonical URL.
+	 */
+	public function maybe_update_paired_url( $redirect_url ) {
+
+		global $wp_rewrite;
+
+		/**
+		 * To address issue AMP version of a comment page.
+		 * Reference: https://github.com/ampproject/amp-wp/issues/6355
+		 */
```

Comment:
A PHP docblock isn't required here; a comment is more suited. I don't think we need to document the issue here either.

```suggestion
```